### PR TITLE
[Event Hubs Trigger] Update template version

### PR DESCRIPTION
**Pull request by jsquire**
# Summary

The focus of these changes is to update the version of the package referenced for the current generation of the Azure SDK.  The version has been bumped to 6.0.0, with the containing folder renamed to reflect the change.

### CODE REVIEW

**Comment by jsquire** [comment]
//cc: @nzthiago, @lilyjma, @hossam-nasr, @soninaren

**Review by soninaren — CHANGES_REQUESTED** [review]
@jsquire , The directory rename would break the build for extension bundles. As each major version of extension bundle is tied to a major version of an extension. We would need to have template for both v5 and v6.

File: Functions.Templates/Templates/EventHubTrigger-CSharp-6.x/.template.config/template.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  "defaultName": "EventHubTriggerCSharp",
+  "postActions": [
+    {
+      "Description": "Adding Reference to Microsoft.Azure.WebJobs.Extensions.EventHubs NuGet package",
+      "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
+      "ContinueOnError": "true",
+      "ManualInstructions": [],
+      "args": {
+        "referenceType": "package",
+        "reference": "Microsoft.Azure.WebJobs.Extensions.EventHubs", "version": "6.0.0",
```
Comment (jsquire) [inline Functions.Templates/Templates/EventHubTrigger-CSharp-6.x/.template.config/template.json:50]:
```suggestion
        "reference": "Microsoft.Azure.WebJobs.Extensions.EventHubs", "version": "6.0.1",
```

**Comment by jsquire** [comment]
@soninaren:  5.x content restored and 6.x line bumped to the latest release.

File: Functions.Templates/Templates/EventHubTrigger-CSharp-6.x/.template.config/template.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  "defaultName": "EventHubTriggerCSharp",
+  "postActions": [
+    {
+      "Description": "Adding Reference to Microsoft.Azure.WebJobs.Extensions.EventHubs NuGet package",
+      "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
+      "ContinueOnError": "true",
+      "ManualInstructions": [],
+      "args": {
+        "referenceType": "package",
+        "reference": "Microsoft.Azure.WebJobs.Extensions.EventHubs", "version": "6.0.1",
```
Comment (jsquire) [inline Functions.Templates/Templates/EventHubTrigger-CSharp-6.x/.template.config/template.json:50]:
```suggestion
        "reference": "Microsoft.Azure.WebJobs.Extensions.EventHubs", "version": "6.0.2",
```

**Comment by liliankasem** [comment]
@soninaren can you take another look please

File: Functions.Templates/Templates/EventHubTrigger-FSharp/.template.config/template.json [hunk 3]
```diff
@@ -51,7 +51,7 @@
       "ManualInstructions": [],
       "args": {
         "referenceType": "package",
-        "reference": "Microsoft.Azure.WebJobs.Extensions.EventHubs", "version": "5.5.0",
+        "reference": "Microsoft.Azure.WebJobs.Extensions.EventHubs", "version": "6.0.1",
```
Comment (jsquire) [inline Functions.Templates/Templates/EventHubTrigger-FSharp/.template.config/template.json:54]:
```suggestion
        "reference": "Microsoft.Azure.WebJobs.Extensions.EventHubs", "version": "6.0.2",

```

File: Functions.Templates/Templates/EventHubTrigger-CSharp-6.x/.template.config/template.json [hunk 4]
```diff
@@ -0,0 +1,71 @@
+{
+  "author": "Microsoft",
+  "classifications": [
+    "Azure Function",
+    "Trigger",
+    "EventHub"
+  ],
+  "name": "EventHubTrigger",
+  "identity": "Azure.Function.CSharp.EventHubTrigger.3.x",
```
Comment (jsquire) [inline Functions.Templates/Templates/EventHubTrigger-CSharp-6.x/.template.config/template.json:9]:
```suggestion
  "identity": "Azure.Function.CSharp.EventHubTrigger.6.x",
```

**Review by soninaren — APPROVED** [review]

**Pull request merged**